server/src/main/java/org/opensearch/ratelimitting/admissioncontrol/settings/IoBasedAdmissionControllerSettings.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.ratelimitting.admissioncontrol.settings;
+
+import org.opensearch.common.settings.ClusterSettings;
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.ratelimitting.admissioncontrol.AdmissionControlSettings;
+import org.opensearch.ratelimitting.admissioncontrol.enums.AdmissionControlMode;
+
+/**
+ * Settings related to IO based admission controller.
+ * @opensearch.internal
+ */
+public class IoBasedAdmissionControllerSettings {
+
+    /**
+     * Default parameters for the IoBasedAdmissionControllerSettings
+     */
+    public static class Defaults {
+        public static final long IO_USAGE_LIMIT = 95;
```

Comment:
Are we finalizing '95' as the initial limit ?

Reply:
Yes for the first monitoring we are finalised on 95 as initial threshold.